doc/release-notes.md

```diff
@@ -110,8 +110,9 @@ Please make sure to move your coins to a wallet with a regular passphrase.
 
 Wallet changes
 --------------
-We continued backporting wallet functionality updates. Most notable changes
-are:
+We continued backporting wallet functionality updates. Most notably, the GUI now
+includes multi-wallet support, which enables working simultaneously with more than
+one wallet file. Other notable changes include:
```

Comment:
Working with multiple wallets in GUI was possible in v17 already, you'd just have to load/unload them via rpc. https://github.com/dashpay/dash/blob/master/doc/release-notes/dash/release-notes-0.17.0.2.md#wallet-changes

Reply:
Hmm, I guess that's a good point... I guess you can drop that commit?